core/math/quick_hull.cpp

```diff
@@ -36,7 +36,7 @@ uint32_t QuickHull::debug_stop_after = 0xFFFFFFFF;
 
 Error QuickHull::build(const Vector<Vector3> &p_points, Geometry::MeshData &r_mesh) {
 
-	static const real_t over_tolerance = 0.0001;
+	real_t over_tolerance = 0.0001;
```

Comment:
Does it make sense to define it here? It looks like the `0.0001` is never used.

I'd remove this one and put `real_t over_tolerance = 3 * UNIT_EPSILON * (aabb.size.x + aabb.size.y + aabb.size.z);` directly under `/* COMPUTE AVAILABLE VERTICES */`.

Reply:
Yeah, that's a good point. I was a bit too happy I found a solution :P